packages/main/src/Link.js

```diff
@@ -102,26 +102,15 @@ const metadata = {
 			defaultValue: WrappingType.None,
 		},
 
-		/**
-		 * Sets the accessible aria name of the component.
-		 *
-		 * @type {String}
-		 * @public
-		 * @since 1.0.0-rc.15
-		 */
-		accessibleName: {
-			type: String,
-		},
-
 		/**
 		 * Receives id(or many ids) of the elements that label the input
 		 *
 		 * @type {String}
 		 * @defaultvalue ""
-		 * @private
-		 * @since 1.0.0-rc.10
+		 * @public
+		 * @since 1.0.0-rc.15
 		 */
-		ariaLabelledby: {
+		accessibleNameRef: {
```

Comment:
Just to make sure this is not by mistake.
 If we keep **accessibleNameRef**, should not we also have "accessibleName", because I can see it's removed?

Reply:
It is not a mistake, this is the agreement we reached with @elenastoyanovaa

Reply:
Ok, thanks!